addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -2361,9 +2361,10 @@ options.registry.CoverProperties = options.Class.extend({
         } else {
             this.$image.css('background-image', `url('${widgetValue}')`);
             this.$target.addClass('o_record_has_cover');
-            const $defaultSizeBtn = this.$el.find('.o_record_cover_opt_size_default');
-            $defaultSizeBtn.click();
-            $defaultSizeBtn.closest('we-select').click();
```

Comment:
Reason for changing this part and not just adding the condition? :thinking:

Reply:
I thought it was better to get the default class directly from the dom than triggering the click on the button but I realize now that it is mandatory to keep the click so that the button displays the right value...
I can only add the condition and let it as it was

Reply:
It's about doing exactly what a click on the select does (it's not only adding a class). Of course, there should be a better way in the future.